envi/archs/arm/disasm.py

```diff
@@ -1488,23 +1500,22 @@ def isDeref(self):
 
     def getOperValue(self, op, emu=None):
         if self.reg == REG_PC:
-            return self.va  # FIXME: is this modified?  or do we need to att # to this?
+            return self.va
```

Comment:
I'm guessing the fixme is no longer valid?

Reply:
correct.